src/software/ai/hl/stp/play/enemy_free_kick/enemy_free_kick_play_fsm.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include "proto/parameters.pb.h"
+#include "software/ai/hl/stp/play/play.h"
+#include "software/ai/hl/stp/tactic/pass_defender/pass_defender_tactic.h"
+
+/**
+ * Play for defending against enemy free kicks
+ */
+
+struct EnemyFreeKickPlayFSM
+{
+    class BlockFreeKicker;
+
+    struct ControlParams
+    {
+        // The maximum allowed speed mode
+        TbotsProto::MaxAllowedSpeedMode max_allowed_speed_mode;
+    };
+
+    DEFINE_PLAY_UPDATE_STRUCT_WITH_CONTROL_AND_COMMON_PARAMS
+
+    static constexpr double HALF_METER_DISTANCE = 0.1;
```

Comment:
Should this be 0.5?

Reply:
good catch, it is supposed to be threshold meters